pkg/machine/qemu/machine.go

```diff
@@ -1078,6 +1082,18 @@ func (v *MachineVM) SSH(_ string, opts machine.SSHOptions) error {
 	return machine.CommonSSH(username, v.IdentityPath, v.Name, v.Port, opts.Args)
 }
 
+func (v *MachineVM) GetConsole() error {
+	f, err := os.Open(v.SerialConsolePath.Path)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	if _, err := io.Copy(os.Stdout, f); err != nil {
+		return err
+	}
```

Comment:
should we copy stdin to it as well to make it interactive or does this not work? Although I guess there in no point because we are unable to login as a user without a static password.

Reply:
Right, the [other console PR](https://github.com/containers/podman/pull/21161) will help enable that.